src/stdlib/String.ts

```diff
@@ -24,5 +24,40 @@ class LCaseCallable extends Callable {
     }
 }
 
+class AscCallable extends Callable {
+    arity = {
+        required: 1,
+        optional: 0
+    }
+
+    call(interpreter: Interpreter, str: BrsString): BrsType {
+        if (str.value.length == 0)
```

Comment:
We might be able to take advantage of some `node`-isms to reduce this a bunch!  Consider something like:

```typescript
return new Int32(
  str.value.charCodeAt(0) || 0
);
```

As it turns out, `"".charCodeAt(0)` returns `NaN`, which lets us fall back to `0` as the base case :smiley:

But it's totally possible that I'm misunderstanding Roku's implementation here!

Reply:
Will switch.